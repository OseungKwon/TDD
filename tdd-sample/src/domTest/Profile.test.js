import React from "react";
import { render, screen } from "@testing-library/react";
import Profile from "./Profile";

describe("<Profile/>", () => {
  it("matches snapshot", () => {
    // render는 반응 요소를 DOM으로 렌더링한다.
    const utils = render(<Profile username="jamong" name="oseung" />);
    expect(utils.container).toMatchSnapshot();
  });
});
