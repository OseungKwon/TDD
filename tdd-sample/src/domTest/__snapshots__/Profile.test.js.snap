// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile/> matches snapshot 1`] = `
<div>
  <div>
    <b>
      jamong
    </b>
    <span>
      (
      oseung
      )
    </span>
  </div>
</div>
`;
